@navikt/core/react/src/tabs/TabList.tsx

```diff
@@ -18,13 +28,121 @@ export type ListType = React.ForwardRefExoticComponent<
 >;
 
 const List = forwardRef<HTMLDivElement, ListProps>(
-  ({ className, ...rest }, ref) => (
-    <TabsList
-      {...rest}
-      ref={ref}
-      className={cl("navds-tabs__tablist", className)}
-    />
-  )
+  ({ className, ...rest }, ref) => {
+    const listRef = useRef<HTMLDivElement | null>(null);
+    const mergedRef = mergeRefs([listRef, ref]);
+    const [displayScroll, setDisplayScroll] = useState({
+      start: false,
+      end: false,
+    });
+
+    const updateScrollButtonState = useCallback(() => {
+      if (!listRef?.current) return;
+      const { scrollWidth, clientWidth } = listRef?.current;
+      let showStartScroll;
+      let showEndScroll;
+
+      const scrollLeft = listRef?.current?.scrollLeft;
+      // use 1 for the potential rounding error with browser zooms.
+      showStartScroll = scrollLeft > 1;
+      showEndScroll = scrollLeft < scrollWidth - clientWidth - 1;
+
+      if (
+        showStartScroll !== displayScroll.start ||
+        showEndScroll !== displayScroll.end
+      ) {
+        setDisplayScroll({ start: showStartScroll, end: showEndScroll });
+      }
+    }, [displayScroll.end, displayScroll.start]);
```

Comment:
Jeg er litt usikker, men jeg tror dette funker (https://reactjs.org/docs/hooks-reference.html#bailing-out-of-a-state-update):
```suggestion
      setDisplayScroll(displayScroll => 
        showStartScroll === displayScroll.start && showEndScroll === displayScroll.end
          ? displayScroll
          : { start: showStartScroll, end: showEndScroll }
        );
    }, []);
```

Reply:
🤩